websub-ballerina/src/websub_advanced_integration_tests/tests/subscriber/test_subscribers_at_basic_auth_secured_hub.bal

```diff
@@ -0,0 +1,169 @@
+    username: "tom",
+    password: "4321"
+});
+
+http:BasicAuthHandler basicAuthHandler2 = new(basicAuthProvider2);
+
+@websub:SubscriberServiceConfig {
+    path: "/websubTwo",
+    subscribeOnStartUp: true,
+    target: "http://localhost:23080/publisherTwo/discover",
+    leaseSeconds: 1200,
+    hubClientConfig: {
+        auth: { authHandler: basicAuthHandler2 },
+        secureSocket: {
+            trustStore: {
+                path: "src/websub_advanced_integration_tests/resources/security/ballerinaTruststore.p12",
+                password: "ballerina"
+            }
+        }
+    }
+}
+service websubSubscriberTwo on websubEP {
+    resource function onNotification (websub:Notification notification) returns @tainted error? {
+        json payload = check notification.getJsonPayload();
+        io:println("WebSub Notification Received by Two: " + payload.toJsonString());
+    }
+}
+
+auth:OutboundBasicAuthProvider basicAuthProvider3 = new({
+    username: "mary",
+    password: "xyz"
+});
+
+http:BasicAuthHandler basicAuthHandler3 = new(basicAuthProvider3);
+
+@websub:SubscriberServiceConfig {
+    path: "/websubThree",
+    subscribeOnStartUp: true,
+    target: "http://localhost:23080/publisher/discover",
+    leaseSeconds: 1200,
+    hubClientConfig: {
+        auth: { authHandler: basicAuthHandler3 },
+        secureSocket: {
+            trustStore: {
+                path: "src/websub_advanced_integration_tests/resources/security/ballerinaTruststore.p12",
+                password: "ballerina"
+            }
+        }
+    }
+}
+service websubSubscriberThree on websubEP {
+    resource function onNotification (websub:Notification notification) returns @tainted error? {
+        json payload = check notification.getJsonPayload();
+        io:println("WebSub Notification Received by Three: " + payload.toJsonString());
+    }
+}
+
+auth:OutboundBasicAuthProvider basicAuthProvider4 = new({
+    username: "tom",
+    password: "1234"
+});
+
+http:BasicAuthHandler basicAuthHandler4 = new(basicAuthProvider4);
+
+@websub:SubscriberServiceConfig {
+    path: "/websubFour",
+    target: "http://localhost:23080/publisherThree/discover",
+    leaseSeconds: 1200,
+    hubClientConfig: {
+        auth: { authHandler: basicAuthHandler4 },
+        secureSocket: {
+            trustStore: {
+                path: "src/websub_advanced_integration_tests/resources/security/ballerinaTruststore.p12",
+                password: "ballerina"
+            }
+        }
+    }
+}
+service websubSubscriberFour on websubEP {
+    resource function onNotification (websub:Notification notification) returns @tainted error? {
+        json payload = check notification.getJsonPayload();
+        io:println("WebSub Notification Received by Four: " + <@untainted>payload.toJsonString());
+        storeOutput(ID_NOTIFICATION_FOUR, "WebSub Notification Received by Four: " + <@untainted>payload.toJsonString());
+    }
+}
+
+@test:Config{
+}
```

Comment:
```suggestion
@test:Config {}
```

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-websub/pull/1/commits/7defc19195f199d3899556e82f9a35eb1ebf4bd5